File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/tablets/LeftTabletActivity.java
package com.jdroid.android.sample.ui.tablets;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class LeftTabletActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return LeftTabletFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/tablets/LeftTabletActivity.kt
package com.jdroid.android.sample.ui.tablets

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class LeftTabletActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return LeftTabletFragment::class.java
    }
}
